### ISSUE TITLE
Font color hardly visible

### ISSUE THREAD

**Issue by MoritzKa** [dhk2/peertube-plugin-lightning#1]
![image](https://user-images.githubusercontent.com/13707997/195884465-104e85c6-4bc0-4ab2-b3fd-09331129538e.png)

**Comment by dhk2** [comment]
Yeah, that one is annoyingly intermittent. I'll try forcing the colors

**Comment by dhk2** [comment]
Testing basic fix now. Eventually need to figure out how to tie into PeerTube's Theme information properly.

**Comment by dhk2** [comment]
Explicitly setting colors on the prompt boxes looks to have resolved the issue. If no one reports the issue reoccurring I'll close this resolved at the end of the week

**Comment by MoritzKa** [comment]
Thanks for fixing this. The new payment modal is another improvement.

**Comment by dhk2** [comment]
By allowing themable colors this issue has come back in a few places :(
